src/pane-element.coffee

```diff
@@ -71,6 +71,8 @@ class PaneElement extends HTMLElement
     hasFocus = @hasFocus()
     itemView = atom.views.getView(item)
 
+    if item.getPath? then $(itemView).addClass('has-file-path')
```

Comment:
Way off topic ...

> we usually use the postfix if for these one-liners.

I've always been curious why everyone does this.  To me it hurts
readability because I can't scan down the left looking for branch points.


On Tue, Apr 14, 2015 at 5:16 PM, Max Brunsfeld <notifications@github.com>
wrote:

> In src/pane-element.coffee
> <https://github.com/atom/atom/pull/6383#discussion_r28384212>:
>
> > @@ -71,6 +71,8 @@ class PaneElement extends HTMLElement
> >      hasFocus = @hasFocus()
> >      itemView = atom.views.getView(item)
> >
> > +    if item.getPath? then $(itemView).addClass('has-file-path')
>
> It'd be nice to use the native DOM API rather than jQuery here, so that we
> can eventually remove our jQuery dependency entirely. Also, we usually use
> the postfix if for these one-liners.
>
> itemView.classList.add('has-file-path') if item.getPath?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/atom/atom/pull/6383/files#r28384212>.
>